daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -7,14 +7,30 @@ description: "Detailed information on the init CLI command"
 
 ## Description
 
-Install Dapr on supported hosting platforms, currently: Kubernetes and self-hosted
+Install Dapr on supported hosting platforms. Supported platforms: Kubernetes and self-hosted
 
 ## Usage
-
 ```bash
 dapr init [flags]
 ```
 
+## Examples
+
+### Initialize Dapr in self-hosted mode
+```bash
+dapr init
+```
+
+### Initialize Dapr in Kubernetes
+```bash
+dapr init -k
+```
+
+### Initialize Dapr in slim self-hosted mode
+```bash
+dapr init -s
```

Comment:
can you provide an example here of installing a particular version of the dapr runtime? use v1.0.0 ;-)

dapr init --runtime-version 1.0.0